west.yml

```diff
@@ -123,7 +123,7 @@ manifest:
       revision: 957d46bc3ce0d5f628f0d525196bb4db207672ee
     - name: nrf_hw_models
       path: modules/bsim_hw_models/nrf_hw_models
-      revision: fec69703cb1ca06fcdab6d5fde01274f0fc5c759
+      revision: 3e9ee62072c8b45a2596ee9e08f12e6c978ae44a
```

Comment:
Please point to the latest: 9e594dace1af29252903938064b8ecb1b8b77678 (no reason to point to older versions)